src/components/Integrations/BigQuery/AuthorizationPage.js

```diff
@@ -59,23 +60,118 @@ const AuthorizationLayout = ({ dependencies: { bigQueryClient, dopplerUserApiCli
+                  </li>
+                  <li>
+                    <a href={_('big_query.url_legacy_control_panel')}>
+                      {_('big_query.plus_configuration')}
+                    </a>
+                  </li>
+                  <li>
+                    <a href={_('big_query.url_legacy_control_panel')}>
+                      {_('big_query.plus_control_panel')}
+                    </a>
+                  </li>
+                  <li>{_('big_query.plus_big_query')}</li>
+                </ul>
+              </nav>
+              <h2>{_('big_query.plus_title')}</h2>
+            </div>
+            <div className="col-lg-7 col-md-7 col-sm-12">
+              <FormattedMessageMarkdown id="big_query.plus_header_MD" />
+            </div>
+          </div>
+          <span className="arrow"></span>
+        </div>
+      </header>
+      <section className="dp-container m-t-60">
+        <div className="dp-rowflex">
+          <div className="col-lg-6 col-md-12 col-sm-12 m-b-24 p-b-42">
+            <div className="dp-content-left">
+              <span>{_('big_query.plus_step_one')}</span>
+              <strong>
+                <div className="p-heading">
+                  <FormattedMessageMarkdown id="big_query.plus_body_step_one_MD" />
+                </div>
+              </strong>
+              <p className="m-t-24">{_('big_query.plus_step_one_paragraph')}</p>
+              <div className="m-t-24">
+                <FormattedMessageMarkdown id="big_query.plus_step_one_paragraph_MD" />
+              </div>
+            </div>
+            <AuthorizationForm emails={data} onSubmit={onSubmit} />
+          </div>
+          <div className="col-lg-6 col-md-12 col-sm-12 m-b-24 p-b-42">
+            <div className="dp-content-right">
+              <span>{_('big_query.plus_step_two')}</span>
+              <strong>
+                <div className="p-heading">
+                  <FormattedMessageMarkdown id="big_query.plus_body_step_two_MD" />
+                </div>
+              </strong>
+              <div className="m-t-24">
+                <FormattedMessageMarkdown id="big_query.plus_step_two_paragraph_MD" />
+              </div>
+              <div className="dp-screen">
+                <img src={dataStudioGif} alt={_('big_query.free_alt_image')} />
+              </div>
+            </div>
+          </div>
+          <div className="col-sm-12 m-b-24">
+            <hr className="dp-h-divider"></hr>
+            <button type="button" className="dp-button button-medium primary-grey m-t-30 m-r-24">
+              {' '}
+              {_('common.cancel')}{' '}
+            </button>
+            <button type="button">
+              <input
+                type="submit"
+                className="dp-button button-medium primary-green m-t-30"
+                form="big-query-configuration-form"
+                value={_('common.save')}
```

Comment:
When is clicked the submit button, the font color is grey. button states should be the same as the Submit Button component. Are you going to do this in another PR?
![image](https://user-images.githubusercontent.com/84402180/132222993-a536ea61-7914-4f3a-a7e8-d06dd5afad4a.png)

Reply:
in next pr, ill resolve this with functionality of cancel button and user mesagges